libplatsupport/plat_include/imx8m/platsupport/plat/serial.h

```diff
@@ -24,7 +25,7 @@
 #if defined(CONFIG_PLAT_IMX8MM_EVK)
 #define DEFAULT_SERIAL_PADDR        UART2_PADDR
 #define DEFAULT_SERIAL_INTERRUPT    UART2_IRQ
-#elif defined(CONFIG_PLAT_IMX8MQ_EVK)
+#elif defined(CONFIG_PLAT_IMX8MQ)
```

Comment:
```suggestion
#elif defined(CONFIG_PLAT_IMX8MQ_EVK) || defined(CONFIG_PLAT_MAAXBOARD)
```
This cannot be generic, because there is no generic board. There might be common guidelines, but in the end it's always up to each specific boards that use a SOC, how things are set up there.